fire-cracker/SendIT-react
# #165096825 Admin can track all Orders

#### What does this PR do?
- Implements  Admin can get all Orders
- Implements  Admin can get all Orders using actions

#### How should this be manually tested?
- Clone repo
- Check out to branch `feature/165096825/admin`
- Run application with `npm run start-dev` in the terminal

#### What are the relevant pivotal tracker stories?
- [Implement Admin can track all orders using react](https://www.pivotaltracker.com/story/show/165096825)
